src/Components/Cases/CaseCard/index.js

```diff
@@ -32,32 +34,34 @@ const CaseCard = (props: Props) => {
         showExpandableButton={true}
       />
       <CardText expandable={true}>
-        <div style={styles.container}>
-          <div style={styles.field}>
-            <span>Parliament: {parliament_case.parliament_session.parliament.name}</span>
-          </div>
-          <div style={styles.field}>
-            <span>Parliament session: {parliament_case.parliament_session.name}</span>
-          </div>
-          <div style={styles.field}>
-            <span>Type: {parliament_case.case_type}</span>
-          </div>
-        </div>
-        <div style={styles.container}>
-          <div style={styles.field}>
-            <span>Status: {parliament_case.case_status}</span>
-          </div>
-          <div style={styles.field}>
-            <span>
-              Created on {moment(parliament_case.created).format('MMM Mo YYYY')}
-            </span>
+        <Paper zDepth={3} rounded={false}>
```

Comment:
would you mind lowering the `zDepth` to 1?
Looks nicer to me :)

### `zDepth = 3`
<img width="1680" alt="screen shot 2017-11-14 at 19 35 01" src="https://user-images.githubusercontent.com/22854722/32797792-1df61a9c-c973-11e7-8a58-a001bcd16829.png">


### `zDepth = 1`
<img width="1680" alt="screen shot 2017-11-14 at 19 35 29" src="https://user-images.githubusercontent.com/22854722/32797794-225c9426-c973-11e7-96da-465f03c3c29a.png">